pkg/storage/utils/filelocks/filelocks.go

```diff
@@ -85,6 +85,10 @@ func acquireLock(file string, write bool) (*flock.Flock, error) {
 		return nil, ErrPathEmpty
 	}
 
+	if _, err = os.Stat(file); err != nil {
+		return nil, err
+	}
```

Comment:
Why do we need to stat the file when we are not using its result?

Reply:
This is only about making sure that the file that is to be locked actually exists. I don't think we want to lock non-existent files.